linode/regions/framework_datasource_schema.go

```diff
@@ -24,49 +24,8 @@ var frameworkDataSourceSchema = schema.Schema{
 		"filter": filterConfig.Schema(),
 		"regions": schema.ListNestedBlock{
 			NestedObject: schema.NestedBlockObject{
-				Attributes: map[string]schema.Attribute{
-					"country": schema.StringAttribute{
-						Description: "The country where this Region resides.",
-						Computed:    true,
-					},
-					"id": schema.StringAttribute{
-						Description: "The unique ID of this Region.",
-						Computed:    true,
-					},
-					"label": schema.StringAttribute{
-						Description: "Detailed location information for this Region, including city, state or region, and country.",
-						Computed:    true,
-					},
-					"site_type": schema.StringAttribute{
-						Description: "The type of this Region.",
-						Computed:    true,
-					},
-					"capabilities": schema.SetAttribute{
-						Description: "A list of capabilities of this Region.",
-						Computed:    true,
-						ElementType: types.StringType,
-					},
-					"status": schema.StringAttribute{
-						Description: "This region’s current operational status.",
-						Computed:    true,
-					},
-				},
-				Blocks: map[string]schema.Block{
-					"resolvers": schema.ListNestedBlock{
-						NestedObject: schema.NestedBlockObject{
-							Attributes: map[string]schema.Attribute{
-								"ipv4": schema.StringAttribute{
-									Description: "The IPv4 addresses for this region’s DNS resolvers, separated by commas.",
-									Computed:    true,
-								},
-								"ipv6": schema.StringAttribute{
-									Description: "The IPv6 addresses for this region’s DNS resolvers, separated by commas.",
-									Computed:    true,
-								},
-							},
-						},
-					},
-				},
```

Comment:
I took this opportunity to reuse the schema & model from the singular data source since their implementations were identical 👍